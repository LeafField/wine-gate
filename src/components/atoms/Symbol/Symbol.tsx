import React from "react";
import Image from "next/image";

export const Symbol = () => {
  return (
    <figure className="symbol absolute inset-0">
      <Image
        src={"/symbol.svg"}
        alt="wine gate"
        unoptimized
        width={124}
        height={124}
        style={{ objectFit: "cover" }}
        priority
        loading="eager"
      />
    </figure>
  );
};
