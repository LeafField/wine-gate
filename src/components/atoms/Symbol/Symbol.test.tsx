import { render } from "@testing-library/react";
import { Symbol } from "./Symbol";

describe("Symbolの単体テスト", () => {
  test("スナップショット", () => {
    const { container } = render(<Symbol />);
    expect(container).toMatchSnapshot();
  });
});
