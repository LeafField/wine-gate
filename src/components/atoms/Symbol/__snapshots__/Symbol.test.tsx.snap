// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Symbolの単体テスト スナップショット 1`] = `
<div>
  <figure
    class="symbol absolute inset-0"
  >
    <img
      alt="wine gate"
      data-nimg="1"
      decoding="async"
      fetchpriority="high"
      height="124"
      loading="eager"
      src="/symbol.svg"
      style="color: transparent; object-fit: cover;"
      width="124"
    />
  </figure>
</div>
`;
